libs/application/template-api-modules/src/lib/modules/shared/data-providers/user-profile/user-profile.module.ts

```diff
@@ -11,14 +11,24 @@ import { BaseTemplateAPIModuleConfig } from '../../../../types'
 import { UserProfileService } from './user-profile.service'
 
 import { Configuration, UserProfileApi } from '@island.is/clients/user-profile'
+import { IslykillService } from '@island.is/api/domains/user-profile'
```

Comment:
We don't want to import from the API Layer. Should be purely using the Client libs.

Reply:
Done